applications/LinearSolversApplication/linear_solvers_define.h

```diff
@@ -0,0 +1,39 @@
+/* KRATOS  _     _                       ____        _
+//        | |   (_)_ __   ___  __ _ _ __/ ___|  ___ | |_   _____ _ __ ___
+//        | |   | | '_ \ / _ \/ _` | '__\___ \ / _ \| \ \ / / _ \ '__/ __|
+//        | |___| | | | |  __/ (_| | |   ___) | (_) | |\ V /  __/ |  \__ |
+//        |_____|_|_| |_|\___|\__,_|_|  |____/ \___/|_| \_/ \___|_|  |___/ Application
+//
+//  Author: Armin Geiser
+*/
+
+
+
+#if !defined(KRATOS_LINEARSOLVERS_DEFINE_H_INCLUDED)
+#define KRATOS_LINEARSOLVERS_DEFINE_H_INCLUDED
+
+// External includes
+#include <Eigen/Core>
+#include <Eigen/Sparse>
+
+namespace Kratos
+{
+
+// IMPORTANT:
```

Comment:
It is the other way round no?
Eigen default is column major while Kratos is row major

Reply:
🙈 fixed the comment